### PR TITLE
router pages for cities implemented

### PR DESCRIPTION
**Pull request by Silvik95**
1)Was added 'publicPath' to resolve the problem with reloading pages
image

2) Implemented new route "/city" to show the clicked city 
image

### CODE REVIEW

**Comment by Silvik95** [comment]
image
temperature added

**Comment by Silvik95** [comment]
Now we can pass any id of the cities in our URL and it will work

File: src/routes/City.jsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+      );
+      const reqDataName = await responseName.json();
+      setCityName(reqDataName.name);
+      const responseInfo = await fetch(
+        `http://localhost:3333/pfa.foreca.com/api/v1/current/${cityId}`
+      );
+      const reqDataInfo = await responseInfo.json();
+      setCityInfo(reqDataInfo.current);
+    })();
+  }, [cityId]);
```
Comment (virzhinia-petkova-redshelf) [inline src/routes/City.jsx]: I'm wondering if you'll be able to use your query hook here

**Review by virzhinia-petkova-redshelf — COMMENTED** [review]
Looks good :) One important comment

File: src/routes/City.jsx (hunk 1 above)
Comment (Silvik95) [reply src/routes/City.jsx]: image
that's how look  my hook logic

File: src/hooks/useFetch.js [hunk 2]
```diff
@@ -1,20 +1,27 @@
...
+import { API_EDPOIONTS, STORAGE_TOKEN } from '../helper/variables';
 
-function useFetch(endpoint, searchQuery) {
+function useFetch(endpoint, searchQuery, id) {
   const [isLoading, setIsLoading] = useState(false);
   const [error, setError] = useState(null);
   const [data, setData] = useState([]);
+  const URL = id
+    ? `http://localhost:3333/pfa.foreca.com/api/v1/${endpoint}/${id}`
+    : `http://localhost:3333/pfa.foreca.com/api/v1/location/${endpoint}/${searchQuery}`;
```
Comment (virzhinia-petkova-redshelf) [inline src/hooks/useFetch.js]: this is not a very extendable way to do it. Think about how you can just pass some params and directly update the url

File: src/hooks/useFetch.js [hunk 3]
```diff
@@ -1,20 +1,27 @@
...
+        const response = await fetch(URL);
         const reqData = await response.json();
-        setData(reqData.locations);
+        if (endpoint === API_EDPOIONTS.SEARCH) {
+          setData(reqData.locations);
+        } else if (endpoint === API_EDPOIONTS.CURRENT) {
+          setData(reqData.current);
+        } else if (endpoint === API_EDPOIONTS.LOCATION) {
+          setData(reqData.name);
+        }
```
Comment (virzhinia-petkova-redshelf) [inline src/hooks/useFetch.js]: what if you just the the data and get the property at the component level - won't it be a lot cleaner and without the endless conditions

**Review by virzhinia-petkova-redshelf — APPROVED** [review]
looks good

**Pull request merged**